src/config/secretKey.js

```diff
@@ -0,0 +1,5 @@
+const secret ={
```

Comment:
Hello @HerveJules It would be better if you use the .env file and add all the environment variables there.

Reply:
thanks brother !